pkg/subsystem/entity/entities.go

```diff
@@ -0,0 +1,24 @@
+// Copyright 2023 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package entity
+
+type Subsystem struct {
+	Name        string
+	PathRules   []PathRule
+	Syscalls    []string
+	Lists       []string
```

Comment:
Can there be more than 1 list? I don't see any such cases in the list you posted.

Reply:
I don't see any such cases.
Let's make it just a string. We make Subsystem==List by design, right?
Also move after the Name.